[Doc-16490] Changed some notes about python gateway service (#16493)
apache/dolphinscheduler: docs/docs/en/guide/installation/standalone.md

bash ./bin/dolphinscheduler-daemon.sh status standalone-server
```

> > Note: Python gateway service is not started along with the api-server by default. If you want to start the Python gateway

Reviewer: ```suggestion
> Note: Python gateway service is disabled by default. If you want to start the Python gateway
```
